jobsdb/jobsdb.go

```diff
@@ -1325,7 +1316,22 @@ func (jd *Handle) createDSInTx(tx *Tx, newDS dataSetT) error {
 	}
 
 	// Create the jobs and job_status tables
-	if _, err = tx.ExecContext(ctx, fmt.Sprintf(`CREATE TABLE %q (
+	if err = jd.createDSTablesInTx(ctx, tx, newDS); err != nil {
+		return err
+	}
+	if err = jd.createDSIndicesInTx(ctx, tx, newDS); err != nil {
+		return err
+	}
```

Comment:
Should we add some error context here?
```suggestion
	if err = jd.createDSTablesInTx(ctx, tx, newDS); err != nil {
		return fmt.Errorf("creating DS tables %w", err)
	}
	if err = jd.createDSIndicesInTx(ctx, tx, newDS); err != nil {
		return fmt.Errorf("creating DS indices %w", err)
	}
```